AdornUI-Stylesheets/card.css

```diff
@@ -0,0 +1,135 @@
+.card__container {
+  width: 25rem;
+  padding: 0.5rem;
+  box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
+  border-radius: 10px;
+}
+.card__image {
+  width: 100%;
+  object-fit: cover;
+}
+.card__title {
+  font-size: 2rem;
+  margin: 0px;
+  color: var(--primary-text-color);
+}
+.card__description {
+  margin: 0px;
+  font-size: 1.2rem;
+  padding: 0.5rem;
+  color: var(--card-text-color);
+  font-weight: 600;
+}
+.product-price-wrapper {
+  width: 50%;
+  justify-content: space-around;
+  padding: 0.5rem;
+}
+.discounted-price,
+.discount {
+  font-size: 1.5rem;
+  font-weight: 700;
+  margin: 0px;
+}
+.discount {
+  color: var(--primary-text-color);
+  font-weight: bold;
+}
+.ratings {
+  background-color: var(--primary-bg-color);
+  padding: 0.5rem;
+  width: fit-content;
+  border-radius: 5px;
+}
+.add-to-wishlist {
```

Comment:
Maybe this file is not the right place as `add-to-wishlist` is not specific to cards. Similarly `remove-from-watchlist`, `add-to-cart`, `discounted-price`, `product-price-wrapper` etc. Cards can be used for a variety of purposes. E-commerce is one example and we this file should be limited to generic cards.